packages/admin-panel/src/pages/resources/PermissionGroupsPage.js

```diff
@@ -21,6 +17,7 @@ const COLUMNS = [
 const CREATE_CONFIG = {
   title: 'Create Permission Group',
   actionConfig: {
+    title: 'Edit Permission Groups',
```

Comment:
I think this one had the wrong pluralisation in the past
```suggestion
    title: 'Edit Permission Group',
```